pagopa/io-app
# fix(IT Wallet): [SIW-1902] Wallet accessibility improvements

## Short description
This PR fixes a small accessibility issue.

> [!NOTE]
> The other accessibility issues mentioned in the Jira ticket must be addressed at the Design System level, in shared components. To speed up the fix, only issues that can be fixed in IO app are included here.

## List of changes proposed in this pull request
- Added proper accessibility label

## How to test
Use a screen reader to navigate the wallet screen with an active wallet instance. The "Cos'è" label should not be announced as "undefined".
